physics/MP/Thompson/mp_thompson.F90

```diff
@@ -858,7 +864,8 @@ subroutine mp_thompson_run(ncol, nlev, con_g, con_rd,        &
                               tprv_rev=tprv_rev, tten3=tten3,                                &
                               qvten3=qvten3, qrten3=qrten3, qsten3=qsten3, qgten3=qgten3,    &
                               qiten3=qiten3, niten3=niten3, nrten3=nrten3, ncten3=ncten3,    &
-                              qcten3=qcten3, pfils=pfils, pflls=pflls)
+                              qcten3=qcten3, pfils=pfils, pflls=pflls,                       &
+                              tc_rain=tc_rain, tc_snow=tc_snow)
```

Comment:
@BinLiu-NOAA Just to clarify, if I set tc_rain = tc_snow = 1.0 in the host model (namelist or hardcoded), then this PR doesn't change the answer, correct?